charts/azimutt/Chart.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v2
+name: azimutt
+description: Explore and optimize any database
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "0.1.0"
```

Comment:
Should this one be sync with https://github.com/azimuttapp/azimutt/blob/main/backend/config/config.exs#L31?
Does it absolutely requires to be changed for any app update? What happen if it doesn't?
Any usual way to handle this other than manually if absolutely required?